Sources/Commands/SwiftPackageTool.swift

```diff
@@ -264,6 +313,14 @@ public class SwiftPackageTool: SwiftTool<PackageMode, PackageToolOptions> {
         print("")
         print("NOTE: Use `swift build` to build packages, and `swift test` to test packages")
     }
+
+    var editUsage: String {
+        let stream = BufferedOutputByteStream()
+        stream <<< "Expected package edit format:\n"
+        stream <<< "swift package edit --name <packageName> (--revision <revision> | --branch <newBranch>)\n"
```

Comment:
I wasn't sure I used this for reference http://docopt.org which says `(A|B)` is either/or which is more fitting here